modules/CONT-CLI-02_Cloning-repository.yml

```diff
@@ -46,10 +46,10 @@ screens:
               id: CONT-CLI-02-create-branch-verification
               success-message: "Great job - you created a branch."
               failure-message: "It looks like you didn't create a branch. Want to try again?"
-        - description: Clone the repository to your desktop.
-          id: CONT-CLI-02-clone
-        - description: Checkout your branch.
-          id: CONT-CLI-02-checkout
+        # - description: Clone the repository to your desktop.
```

Comment:
this is only a temporary fix, it seems that the importer reads steps through twice since there is both verified and unverified steps, currently that is an assumption but there is a story open for the importer to be looked at